packages/lib/event-types/transformers/api-request.ts

```diff
@@ -149,6 +149,8 @@ const SystemFieldsSchema = z.object({
   defaultLabel: z.string(),
   labe: z.string().optional(),
   editable: z.enum(["system-but-optional", "system"]),
+  minLength: z.number().optional(),
+  maxLength: z.number().optional(),
```

Comment:
@calcom/platform Is it possible to reuse `packages/features/form-builder/schema.ts` somehow here?